src/gui/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 project(gui)
 find_package(Qt5 REQUIRED COMPONENTS Widgets Svg Qml Quick)
+if (Qt5Widgets_VERSION VERSION_LESS 5.9.0 
```

Comment:
I think checking for one version should be enough. I don't think mixing versions is a common (or even valid) use case

Reply:
Fair enough, simplicity may indeed be more valuable here. Adressed in 9239e60